src/test/kotlin/pl/droidsonroids/gradle/jacoco/testkit/JaCoCoTestKitPluginFunctionalTest.kt

```diff
@@ -16,7 +16,7 @@ class JaCoCoTestKitPluginFunctionalTest {
 
     @Before
     fun setUp() {
-        temporaryFolder.newFile("gradle.properties").fillFromResource("testkit-gradle.properties")
+        temporaryFolder.newFile("gradle.properties")
```

Comment:
💡 I think I now know what this was, it's the "intergration" with jacoco! The file is not generated because I uncommented the plugin in `build.gradle`